Extension/package.json

```diff
@@ -1201,6 +1201,11 @@
                 },
                 "default": []
               },
+              "envFile": {
+                "type": "string",
+                "description": "Environment variables passed to the program by a file.",
```

Comment:
Should there be more description about the format of the file here, or are users expected to know what this file is?

Reply:
We will need to update the vscode docs or add to launch.md with a similar description in https://code.visualstudio.com/docs/python/environments#_environment-variable-definitions-file.

Any preference?

Reply:
No preference. If you move it to the vscode docs, you may want to mention that in the description here.

Reply:
Added to launch.md